packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -968,6 +968,28 @@ Specifies a particular platform or device to match on
 }
 ```
 
+#### Device Condition object
+
+<ApiLifecycle access="ie" /><br>
+
+Specifies the device condition to match on
+
+| Parameter | Description              | Data Type | Required |
+| ---       | ---                      | ---       | ---      |
+| registered   | If the device is registered | Boolean     | No      |
+| managed   | If the device is managed | Boolean     | No      |
+
+> **Note:** When managed is set, registered is required, and the value must be true.
```

Comment:
```suggestion
> **Note:** When `managed` is `TRUE`, `registered` is required and must be `TRUE`.
```

Reply:
what I mean is that if you specify `manage` attribute, not matter the value is `TRUE` or `FALSE`, the `register` must be `TRUE`. The following is not allowed:
{
"managed": true
}
or
{
"managed": false
}

Reply:
Updated my suggestion. How's that?

Reply:
sounds good. thanks!